### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by nnat425**
- Text for no javascript error message is now not uppercase 
- Fix async check issue when e-mail is already verified 
- Add some logging

### CODE REVIEW

File: app/assets/javascripts/sign_up_async_request.js.erb [hunk 1]
```diff
@@ -78,6 +78,21 @@ function handleError(){
 	$("#check_email_span").text("We've encountered an error. Please try again later or email help@mylibrarynyc.org for assistance.");
 }
 
+function validEmailMessage(){
```
Comment (LucasCioffi) [inline app/assets/javascripts/sign_up_async_request.js.erb]: `showValidEmailMessage` might be a bit more accurate for what it does.

Comment (LucasCioffi) [reply app/assets/javascripts/sign_up_async_request.js.erb]: Maybe pass in a boolean value about whether the validation passed or failed, and then you can fully control the disabling of the form's submit button here.

File: app/models/user.rb [hunk 2]
```diff
@@ -123,6 +123,12 @@ def send_request_to_patron_creator_service
         }
       ],
     }
+    Rails.logger.debug(
+       {
+        'status' => 'Request send to patron creator service',
+        'dataSent' => query
```
Comment (daryanypl) [inline app/models/user.rb]: I'd rather not send the query to the log without passing it through a safety filter first, but we can fix this after we erplace the logging with our wrapper.

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -53,7 +52,7 @@
   </div>
 
   <div class="field-group">
-    <%= label_tag :user_school_id, 'Your school:', :id => 'userSchool-label' %>
+    <%= label_tag :user_school_id, 'Your school', :id => 'userSchool-label' %>
```
Comment (daryanypl) [inline app/views/devise/registrations/new.html.erb]: Using a label for id info might be problematic for accessibility, but let's write this down and move forward and do accessibility when we have Willa.

File: app/models/user.rb (hunk 2 above)
Comment (nnat425) [reply app/models/user.rb]: I agree.

**Pull request merged**